cmd/influxd/run/config_test.go

```diff
@@ -141,6 +141,8 @@ bind-address = ":2010"
 [[udp]]
 bind-address = ":4444"
 
+[[udp]]
```

Comment:
Should it be possible to set the second UDP listener with environment variables without needing an empty section here? I think that most people who would use environment variables to configure InfluxDB probably wouldn't use a configuration file, otherwise they would just use the configuration file.